src/views/v2/widgets/widget-events-list/event/date.php

```diff
@@ -19,14 +19,20 @@
 
 $event_date_attr = $event->dates->start->format( Dates::DBDATEFORMAT );
 
+if ( $event->multiday ) {
+	// The date returned back contains HTML and is already escaped.
+	$event_date = $event->schedule_details->value();
+} else if ( $event->all_day ) {
```

Comment:
:warning: Usage of ELSE IF is discouraged; use ELSEIF instead

<sub>hash: 4a741d7d9ee530e81f3c09080fc7ffcf</sub>